Makefile

```diff
@@ -89,6 +89,7 @@ ci=
 cr_deployment_only=false
 load_default_config=false
 container_engine=
+cluster=openshift
```

Comment:
Just a minor observation: we have a function in `client` package to discover in which cluster we are. Not sure if your intention is to run this command telling explicit to the test framework where you are, but I think you could leverage from this function instead and run Kubernetes specifics when `client.IsOpenShift == false`.

Reply:
Will try to run the tests using "client.NewForConsole().IsOpenshift()" and everything goes as expected, I will get rid of this new cluster parameter. Thanks!

Reply:
I've just tested in K8s and Openshift and the above works. I've already removed this new cluster argument.